main.py

```diff
@@ -335,6 +318,9 @@ async def on_message(message: discord.Message):
     LOGGER.info("End of message listener stack.")
 
     LOGGER.info("Processing additional commands.")
+
+    # filter out potential drone IDs so they can use commands as well
+    message.content = re.match(r"(?:\d{4} :: )?(.*)", message.content).group(1)
```

Comment:
Gave it more t\*\*\*ght. Better to process this within the prepending module instead.